node/external/nodeApiResolver.go

```diff
@@ -73,7 +73,7 @@ func (nar *NodeApiResolver) Close() error {
 		err2 = nar.vmFactory.Close()
 	}
 	if err1 != nil || err2 != nil {
-		return fmt.Errorf("err closing vmContainer: %w, err closing vmFactory: %w", err1, err2)
+		return fmt.Errorf("err closing vmContainer: %v, err closing vmFactory: %v", err1, err2)
```

Comment:
is this ok? at least use `%s` and `errx.String()`

Reply:
no, will panic if err is nil. Works as expected as it is

Reply:
ok. tested locally and the error printing is the same.